### DISCUSSION TITLE
VueJS computed properties in Trame?

### DISCUSSION THREAD

**Discussion by DavidBerger98** [Kitware/trame#122] [Q&A] [answered]
I found myself writing very long strings in `v_for` or `v_if` statements to create specific lists in my application for things that could be [computed properties](https://vuejs.org/guide/essentials/computed.html) instead.

Is there a possibility to define computed properties in Trame that would be client-only?

**Comment by jourdain** [comment, marked as answer]
We could make that possible but having them computed on the server with some `@state.change('a', 'b', 'c')` logic.

**Comment by jourdain** [comment]
But we could create some declarative vue components to update those computed / client-only properties.
